torch/onnx/symbolic_opset11.py

```diff
@@ -93,25 +93,37 @@ def pixel_shuffle(g, self, upscale_factor):
 
 
 def _interpolate(name, dim, interpolate_mode):
-    def symbolic_fn(g, input, output_size, align_corners=None):
+    def symbolic_fn(g, input, output_size, *args):
+        scales, align_corners = sym_help._get_interpolate_attributes(g, interpolate_mode, args)
         align_corners = sym_help._maybe_get_scalar(align_corners)
         coordinate_transformation_mode = "asymmetric" if interpolate_mode == "nearest" \
             else "align_corners" if align_corners else "pytorch_half_pixel"
         empty_tensor = g.op("Constant", value_t=torch.tensor([], dtype=torch.float32))
-        input_size = input.type().sizes()
-        input_size = g.op("Constant", value_t=torch.tensor(input_size[0:2], dtype=torch.int64))
-        output_size = g.op("Cast", output_size, to_i=sym_help.cast_pytorch_to_onnx["Long"])
-        output_size = g.op("Concat", input_size, output_size, axis_i=0)
 
-        return g.op("Resize",
-                    input,
-                    empty_tensor,  # roi only takes effect with coordinate_transformation_mode="tf_crop_and_resize"
-                    empty_tensor,  # scales is not needed since we are sending out_size
-                    output_size,
-                    coordinate_transformation_mode_s=coordinate_transformation_mode,
-                    cubic_coeff_a_f=-0.75,  # only valid when mode="cubic"
-                    mode_s=interpolate_mode,  # nearest, linear, or cubic
-                    nearest_mode_s="floor")  # only valid when mode="nearest"
+        if scales is None:
+            input_size = g.op("Shape", input)
+            input_size_beg = sym_help._slice_helper(g, input_size, axes=[0], ends=[2], starts=[0])
+            output_size = g.op("Cast", output_size, to_i=sym_help.cast_pytorch_to_onnx["Long"])
+            output_size = g.op("Concat", input_size_beg, output_size, axis_i=0)
```

Comment:
It seems input_size_beg here is int32, while output_size is int64, can you confirm? @lara-hdr . I'm having a problem converting ONNX Resize op to TVM, because TVM's concat op assumes input dtypes are all same.